src/server/internalServer.cpp

```diff
@@ -223,7 +223,8 @@ typedef std::set<std::string> Languages;
 Languages getLanguages(const Library& lib, const Library::BookIdSet& bookIds) {
   Languages langs;
   for ( const auto& b : bookIds ) {
-    langs.insert(lib.getBookById(b).getLanguage());
+    const auto bookLangs = lib.getBookById(b).getLanguages();
+    langs.insert(bookLangs.begin(), bookLangs.end());
```

Comment:
This seems a functional change, not just a deprecation of `getLanguage` so it should go in another commit.
(Not important in itself, but as we have other changes to do, let's split this change correctly)

Reply:
Done